docker/scripts/extract_raw_thumbnail.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+echo Converting and shrinking image...
+/dcraw/dcraw -c /tmp/imagefile | convert - -resize 640x360 -define modulate:colorspace=HSB -modulate 106,90,96\> /tmp/resized.jpg > /tmp/logfile 2>&1
```

Comment:
please install dcraw to /usr/local/bin, not root

Reply:
Done.

Reply:
This did not work in AWS Docker, likely because of a compiler error. I have fixed this now by using the supplied binary.